src/families/bitcoin/libcore-broadcast.js

```diff
@@ -2,12 +2,17 @@
 
 import { patchOperationWithHash } from "../../operation";
 import { makeBroadcast } from "../../libcore/broadcast";
+import invariant from "invariant";
 
 async function broadcast({
+  core,
   coreAccount,
   signedOperation: { signature, operation }
 }) {
-  const bitcoinLikeAccount = await coreAccount.asBitcoinLikeAccount();
+  const bitcoinLikeAccount = await new core.Bitcoin().fromCoreAccount(
```

Comment:
i don't think this will work on mobile. there is no instantiation possible over react native. (unless you generated some JS somewhere?)

```
core.Bitcoin.fromCoreAccount
```

doesn't work?

Reply:
IIRC, nah, it doesn’t work, because `Bitcoin` is never created / inserted as an empty object in `core`. But maybe there’s something simple to create the object?

Reply:
i don't understand why you can do `core.CoreEthereumLikeAccount.fromCoreAccount` in that same PR

Reply:
I think it’s a mistake. I’m patching all this up.